packages/paste-website/src/pages/components/anchor/index.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Anchor
+description: The anchor can be used to hyperlink to another URL. It accepts both internal and external URLs.
+---
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<p>{props.pageContext.frontmatter.description}</p>
+
+<p>Example use cases may include:</p>
+
+<ul>
+  <li>Linking to another page within an app or website</li>
+  <li>Linking to an external domain outside of the current app or website</li>
+</ul>
+
+<p>Anchors can only include text.</p>
```

Comment:
According to the typings, it accepts React.ReactNode.  We should allow icons for example.

Reply:
I'll remove this line for now.